### PR TITLE
Add Plant Images

### PR DESCRIPTION
**Pull request by Drewbi**
Picked 2-4 images for each plant

### CODE REVIEW

File: components/PlantInfo.vue [hunk 1]
```diff
@@ -23,15 +23,16 @@
     </v-btn>
     <v-card flat>
       <v-carousel
+        v-if="plantInfo.images.length > 0"
```
Comment (JeremiahPinto) [inline components/PlantInfo.vue]: use v-show instead of v-if. v-if should only be used if there's multiple conditions. Plus v-show is faster

**Comment by JeremiahPinto** [comment]
Why do some files not have a new line at the end?

**Comment by Drewbi** [comment]
@JeremiahPinto Have you looked at all the images we chose? They are very informative :)

**Comment by Drewbi** [comment]
Superseded by #81

**Pull request closed without merging**